examples/tutorials/plot.py

```diff
@@ -54,7 +54,7 @@
 fig.plot(
     x=data.longitude,
     y=data.latitude,
-    sizes=0.02 * (2 ** data.magnitude),
+    size=0.02 * (2 ** data.magnitude),
     style="cc",
     color="white",
     pen="black",
```

Comment:
Please also change 'sizes' to 'size' at Line 66 in this `examples/tutorials/plot.py` file.

Reply:
Similarly, at line 66, the sentence is about the symbol sizes, not the `size` parameter.

Reply:
The full paragraph from L64-66 reads:

```
# Notice that we didn't include the size in the ``style`` parameter this time, just the
# symbol ``c`` (circles) and the unit ``c`` (centimeter). So in this case, the sizes
# will be interpreted as being in centimeters.
```

So either change the `size` in L64 to `sizes`, or the `sizes` in L66 to `size`?